### PR TITLE
Fix disqus google analytics integration

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @mauro-oto,

This PR fixes the disqus and GA integration.

Please check it out.

Thanks!

### CODE REVIEW

File: _site/authors/etagwerker/index.html [hunk 1]
```diff
@@ -115,7 +115,7 @@ <h2 class="post-title"><a class="read-more" href="/blog/software%20development/s
     <script type="text/javascript">
 
       var _gaq = _gaq || [];
-      _gaq.push(['_setAccount', '<%= ENV['GOOGLE_ANALYTICS_ID'] %>']);
+      _gaq.push(['_setAccount', '']);
```
Comment (mauro-oto) [inline _site/authors/etagwerker/index.html]: @etagwerker Looks like it's not populating the ID for `authors/:author/index.html`.

Comment (etagwerker) [inline _site/authors/etagwerker/index.html]: @mauro-oto Weird. I'll look into it.

**Comment by etagwerker** [comment]
@mauro-oto Done! I don't know why I hadn't regenerated the authors index page.

**Comment by mauro-oto** [comment]
@etagwerker there's something odd about this, as now the non-author pages have lost the GA id. All of them seem to use the same `default.html` layout, so I'm not sure what's wrong.

**Comment by etagwerker** [comment]
@mauro-oto Weird. I'll look into it.

**Comment by etagwerker** [comment]
@mauro-oto Now it seems to be alright. I regenerated it one more time.

**Pull request merged**